packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -115,14 +161,20 @@ class AccountImportOptions extends Component {
     });
   };
 
-  hasExistingAddressForImport = (addressForImport, chainId) => {
+  isCurrentChainIdTheAddressForImportChainId = chainIdInt => {
+    const { chainId: currentChainIdBN } = this.props;
+
+    return BigNumber(chainIdInt).eq(currentChainIdBN);
+  };
+
+  hasExistingAddressForImport = (addressForImport, chainIdInt) => {
```

Comment:
not related to this PR: it took me a while to understand what the function does, name is a bit confusing. maybe rename to `accountAlreadyExists` ?

Reply:
i've addressed this in commit cab245e25f448951ef01d25f6465b9c88ce8d363